### ISSUE TITLE
ENHANCEMENT - Task Group assigned to roles - Include policy complete_all = false

### ISSUE THREAD

**Issue by atl2017** [smap-consulting/smapserver#499]
### Expected Behaviour: Currently when you create a Task Group based on results you can assign it to roles (multiple users). Tasks are displayed to all users with Role. 
It could be interesting to include a policy that when one user complete the task, that tasks won´t be longer displayed to the users. Now, the only way to use it is just to complet for one user and the rest have to "reject" task manually.

In summary: Have two policies. Current: complete_all = yes, New policy: complete_all = false

![image](https://user-images.githubusercontent.com/29519901/62210876-caaccd00-b362-11e9-8f7e-8bc909ebe5d0.png)

**Comment by atl2017** [comment]
Great feature. Works fine.
